packages/inventory/src/Stocktake/DetailView/AppBarButtons.tsx

```diff
@@ -19,6 +27,13 @@ export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({
   const isDisabled = useIsStocktakeDisabled();
   const { OpenButton } = useDetailPanel();
   const t = useTranslation('common');
+  const { data } = useStocktake();
+  const { print, isPrinting } = usePrintReport();
```

Comment:
this is really `downloadReport` aka `getAPrintableVersionOfThisThingy`
it's not ideal for a user.. but ok?
I experimented with `window.print` - and that's fine for an HTML page, but printing a PDF in chrome made the browser crash 😢

Reply:
Yeah I tried to communicate that this endpoint using the `report` terminology is not correct and is confusing (my suggestion for gql I think was `documentToFile`

So.. as is software developer tradition, my naming is better than everyones! `useDocumentAsFile`... 

🤷 

`usePrintReport` lgtm

Reply:
it is better described as `useDocumentAsFile`, and you're right, I've picked up on the api terminology and used that without thinking it through. I see this all as temporary ( or hope that it is anyway.. )